launcher/test-framework/org.jboss.tools.common.launcher.reddeer/src/org/jboss/tools/common/launcher/reddeer/wizards/NewLauncherProjectWizardPage.java

```diff
@@ -0,0 +1,195 @@
+	}
+	
+	/**
+	 * Sets target mission
+	 * @param targetMissionName
+	 */
+	public void setTargetMission(String targetMissionName) {
+		new DefaultCombo(0).setSelection(targetMissionName);
+	}
+
+	/**
+	 * Sets nth target mission.
+	 * @param index
+	 */
+	public void setTargetMission(int index) {
+		new DefaultCombo(0).setSelection(index);
+	}
+	
+	/**
+	 * Sets target runtime.
+	 * @param targetRuntimeName
+	 */
+	public void setTargetRuntime(String targetRuntimeName) {
+		new DefaultCombo(1).setSelection(targetRuntimeName);
+	}
+	
+	/**
+	 * Sets nth target runtime.
+	 * @param index
+	 */
+	public void setTargetRuntime(int index) {
+		new DefaultCombo(1).setSelection(index);
+	}
+
+	/**
+	 * Toggles project path checkbox
+	 * @param newCheckboxState
+	 */
+	public void toggleTheCheckBox(boolean newCheckboxState){
+		CheckBox ch = new CheckBox("Use default location");
+		ch.toggle(newCheckboxState);
+	}
+	
+	/**
+	 * Sets the "Location" value
+	 * @param newProjectLocation
+	 */
+	public void setCustomProjectLocation(String newProjectLocation){
+		LabeledText projectLocationLabeledText = new LabeledText("Location:");
+		projectLocationLabeledText.setText(newProjectLocation);
+	}
+	
+	/**
+	 * Returns the current project location path
+	 * @return
+	 */
+	public String getProjectLocation(){
+		LabeledText projectLocationLabeledText = new LabeledText("Location:");
+		return (projectLocationLabeledText.getText());
+	}
+	
+	/**
+	 * Sets the "Project name" value
+	 * @param newProjectName
+	 */
+	public void setProjectName(String newProjectName){
+		LabeledText projectLabeledText = new LabeledText("Project name:");
+		projectLabeledText.setText(newProjectName);
+	}
+	
+	/**
+	 * Sets the "Artifact id" value
+	 * @param newArtifactId
+	 */
+	public void setArtifactId(String newArtifactId) {
+		LabeledText projectLocationLabeledText = new LabeledText("Artifact id:");
+		projectLocationLabeledText.setText(newArtifactId);
+	}
+	
+	/**
+	 * Returns the "Artifact id" value
+	 * @return
+	 */
+	public String getArtifactId() {
+		LabeledText projectLocationLabeledText = new LabeledText("Artifact id:");
+		return (projectLocationLabeledText.getText());
+	}
+	
+	/**
+	 * Sets the "Group id" value
+	 * @param newGroupId
+	 */
+	public void setGroupId(String newGroupId) {
+		LabeledText projectLocationLabeledText = new LabeledText("Group id:");
+		projectLocationLabeledText.setText(newGroupId);
+	}
+	
+	/**
+	 * Returns the "Group id" value
+	 * @return
+	 */
+	public String getGroupId(){
+		LabeledText projectLocationLabeledText = new LabeledText("Group id:");
+		return (projectLocationLabeledText.getText());
+	}
+	
+	/**
+	 * Performs the click action on the "Finish" button
+	 */
+	public void clickTheFinishButton() {
+		Button finishButton = new PushButton("Finish");
+		finishButton.click();
+	}
+	
+	/**
+	 * Returns if the "Finish" button is enabled
+	 * @return
+	 */
+	public boolean isTheFinishButtonEnabled() {
```

Comment:
@zcervink I would remove "The" from method names, it makes methods names longer without adding any value.

Reply:
removed.